src/analyserqueue.cpp

```diff
@@ -26,17 +26,23 @@
 // 100 for 10% step after finalize
 #define FINALIZE_PERCENT 1
 
-// We need to use a smaller block size becuase on Linux, the AnalyserQueue
-// can starve the CPU of its resources, resulting in xruns.. A block size of
-// 8192 seems to do fine.
-const int kAnalysisBlockSize = 8192;
+namespace
+{
+    const Mixxx::AudioSource::size_type kAnalysisChannels = 2; // stereo
+
+    // We need to use a smaller block size because on Linux, the AnalyserQueue
+    // can starve the CPU of its resources, resulting in xruns. A block size of
+    // 4096 samples per channel seems to do fine.
+    const Mixxx::AudioSource::size_type kAnalysisFrameCount = 4096;
```

Comment:
*Count makes me think it is counting. It is hard to find a short and expressive name here :-/
Probably "kAnlysisBlockSizeFrames" and   "kAnlysisBlockSizeSamples"